runtime-attach/README.md

```diff
@@ -10,10 +10,27 @@ The attachment will _not_ be initiated in the following cases:
 * The attachment is not requested from the _main_ thread
 * The attachment is not requested from the `public static void main(String[] args)` method
 * The agent is already attached
-* The application is running on a JRE (a JDK is necessary)
-* The application is running on a read-only file system
+* The application is running on a JRE (a JDK is necessary, see [this issue](https://github.com/raphw/byte-buddy/issues/374))
+* The temporary directory should be writable
+
+To use runtime attachment, you have to add the following dependency:
```

Comment:
check out doc pattern used in instrumentation repo for documenting how to add dependencies: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/okhttp/okhttp-3.0/library/README.md#add-these-dependencies-to-your-project